Generate.py

```diff
@@ -161,6 +169,10 @@ def main(args=None) -> Tuple[argparse.Namespace, int]:
                  f"{seed_name} Seed {seed} with plando: {args.plando}")
 
     if not weights_cache:
+        if player_errors:
+            errors = '\n\n'.join(player_errors)
+            raise Exception(f"Encountered {len(player_errors)} error(s) in player files. "
```

Comment:
These two should probably also be ValueError? This one at least, I'm less confident about the one below.